reconcile/dashdotdb_dvo.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import os
+import requests
+from urllib.parse import urljoin
+
+import reconcile.utils.threaded as threaded
+import reconcile.queries as queries
+from reconcile.utils.oc import OC_Map
+from reconcile.utils.oc import StatusCodeError
+from reconcile.utils.secret_reader import SecretReader
+
+LOG = logging.getLogger(__name__)
+QONTRACT_INTEGRATION = 'dashdotdb-dvo'
+DASHDOTDB_SECRET = os.environ.get('DASHDOTDB_SECRET',
+                                  'app-sre/dashdot/auth-proxy-production')
+
+
+class DashdotdbDVO:
+    def __init__(self, dry_run, thread_pool_size):
+        self.dry_run = dry_run
+        self.thread_pool_size = thread_pool_size
+        self.settings = queries.get_app_interface_settings()
+        secret_reader = SecretReader(settings=self.settings)
+        secret_content = secret_reader.read_all({'path': DASHDOTDB_SECRET})
+        self.dashdotdb_url = secret_content['url']
+        self.dashdotdb_user = secret_content['username']
+        self.dashdotdb_pass = secret_content['password']
+        self.logmarker = "DDDB_DVO:"
+
+    def _post(self, deploymentvalidation):
+        if deploymentvalidation is None:
+            return None
+        cluster = deploymentvalidation['cluster']
+        dvdata = deploymentvalidation['data']
+        LOG.info('%s posting validations for %s', self.logmarker, cluster)
```

Comment:
I would do this a debug message or this integration logs are going to be very verbose 😅

Reply:
agreed. will fix. ..and gives me an idea for backlogging: perhaps exposing some runtime activity metrics would be good.

Reply:
changed to debug.